src/QMCWaveFunctions/Fermion/MatrixUpdateOMPTarget.h

```diff
@@ -197,6 +199,69 @@ class MatrixUpdateOMPTarget
       grad_now[iw] = {grads_value_v[iw][0], grads_value_v[iw][1], grads_value_v[iw][2]};
   }
 
+  template<typename GT>
```

Comment:
Testing with this should be added in test_DiracDeterminantBatched.cpp.  That test is ready for testing the mw_ methods.
An implementation of this in MatrixDelayedUpdateCUDA is needed, once the test is implemented that should be some pretty straight forward porting that I could help with.

Reply:
Yes I agree we need the testing. This function will be called by DiracDeterminantBatched, and there is already a test in place. I just need to comment out the corresponding sections in the existing unit test.  (see  [here](https://github.com/QMCPACK/qmcpack/blob/develop/src/QMCWaveFunctions/tests/test_DiracDeterminantBatched.cpp#L784-L795)  and [here](https://github.com/QMCPACK/qmcpack/blob/develop/src/QMCWaveFunctions/tests/test_DiracDeterminantBatched.cpp#L815-L819))